backend/services/uploadService.js

```diff
@@ -1,22 +1,47 @@
-var userRepository = require('../repositories/userRepository');
+var userRepository = require('../repositories/userRepository'),
+    fs = require('fs');
+
 var rootPath = 'C:/www/BSA-16-ngCamp/test/';
 
 function UploadService() {
 }
 
 UploadService.prototype.saveFile = saveFile;
 
+
+
+function checkDir(path){

```

Comment:
It should be implemented to create each chunk of path and create new in other case